frontend/src/features/mapSideBar/hooks/useBuildingApi.tsx

```diff
@@ -0,0 +1,60 @@
+import { request, success, error } from 'actions/genericActions';
+import { IBuilding } from 'actions/parcelsActions';
+import { showLoading, hideLoading } from 'react-redux-loading-bar';
+import CustomAxios, { LifecycleToasts } from 'customAxios';
+import { ENVIRONMENT } from 'constants/environment';
+import * as actionTypes from 'constants/actionTypes';
+import * as pimsToasts from 'constants/toasts';
+import * as API from 'constants/API';
+
+const buildingCreatingToasts: LifecycleToasts = {
+  loadingToast: pimsToasts.building.BUILDING_CREATING,
+  successToast: pimsToasts.building.BUILDING_CREATED,
+  errorToast: pimsToasts.building.BUILDING_CREATING_ERROR,
+};
+
+const buildingUpdatingToasts: LifecycleToasts = {
+  loadingToast: pimsToasts.building.BUILDING_UPDATING,
+  successToast: pimsToasts.building.BUILDING_UPDATED,
+  errorToast: pimsToasts.building.BUILDING_UPDATING_ERROR,
+};
+
+export const useBuildingApi = () => {
```

Comment:
Document these calls to make it obvious what these endpoints do from intellisense

Reply:
doing that now.